test/test.h

```diff
@@ -23,6 +23,7 @@
 #include <gtest/gtest.h>
 
 #include <boost/algorithm/string/replace.hpp>
+#include <boost/format.hpp>
```

Comment:
OSX was again failing with some gurka tests not including boost/format.hpp.. let's just do it here and forget about it..